spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/Topology.scala

```diff
@@ -18,30 +18,24 @@ package com.intel.analytics.bigdl.nn.keras
 
 import com.intel.analytics.bigdl.nn.Graph._
 import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, Activity}
-import com.intel.analytics.bigdl.nn.{Graph, GraphSerializable, StaticGraph, Sequential => TSequential}
+import com.intel.analytics.bigdl.nn.{Container, Graph, GraphSerializable, StaticGraph, Sequential => TSequential}
 import com.intel.analytics.bigdl.serialization.Bigdl.BigDLModule
 import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Shape
 import com.intel.analytics.bigdl.utils.serializer._
-import com.intel.analytics.bigdl.utils.{Shape, Util}
 
+import scala.collection.JavaConverters._
 import scala.reflect.ClassTag
 
 class Model[T: ClassTag](private val _inputs : Seq[ModuleNode[T]],
       private val _outputs : Seq[ModuleNode[T]])(implicit ev: TensorNumeric[T])
-  extends StaticGraph[T](_inputs, _outputs, None, false) {
-
-  Util.excludeNotKeras(inputs.map(_.element))
-  Util.excludeNotKeras(outputs.map(_.element))
+  extends StaticGraph[T](_inputs, _outputs, None) {
 
   this.inputShapeValue = Shape(inputs.map{n => n.element.getInputShape()}.toList)
 
-  this.outputShapeValue = Array(outputs.map{_.element.getOutputShape()}: _*)
-
-  isBuilt = true
-
-  override private[bigdl] def isCompatibleWithKeras(): Boolean = true
+  this.outputShapeValue = Shape(outputs.map{_.element.getOutputShape()}.toList)
```

Comment:
What if inputs or outputs are not Keras style?

Reply:
Would add unittest for that.